docs/gallery/interactive/binaural_synthesis.ipynb

```diff
@@ -0,0 +1,327 @@
+    "hrir_file = os.path.join(pf.signals.files.file_dir, files[0])\n",
+    "\n",
+    "# load a SOFA file containing the HRIR dataset\n",
+    "hrirs, sources, _ = pf.io.read_sofa(hrir_file)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we are loading the included HRIR dataset from the FABIAN dummy head by [Brinkmann _et al._](https://depositonce.tu-berlin.de/items/3b423df7-a764-4ce1-9065-4e6034bba759).\n",
+    "`pyfar` includes a [method to load specific HRIRs from the dataset](https://pyfar.readthedocs.io/en/stable/modules/pyfar.signals.files.html#pyfar.signals.files.head_related_impulse_responses) but the example shown here is the general approach for loading a SOFA file.\n",
+    "\n",
+    "First, we will plot all possible source locations that are contained in the dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sources.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see that the included dataset only contains a limited number of source positions - namely, the horizontal plane and the median plane.\n",
+    "In practice, an HRIR dataset usually contains more source positions covering the whole sphere around the listener.\n",
+    "\n",
+    "Using the `pyfar` coordinates, <!-- LINK TODO --> a specific direction can be selected from the dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# define the direction of the desired sound source; here, we choose the right direction\n",
+    "elevation = 0\n",
+    "azimuth = -90\n",
+    "\n",
+    "# convert to radians\n",
+    "elevation = elevation * np.pi / 180\n",
+    "azimuth = azimuth * np.pi / 180\n",
+    "\n",
+    "desired_direction = pf.Coordinates.from_spherical_elevation(\n",
+    "    azimuth, elevation, 2\n",
+    ")\n",
+    "\n",
+    "index, _ = sources.find_nearest(desired_direction)\n",
+    "\n",
+    "sources.show(index)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the plot, the selected direction is marked with a red dot.\n",
+    "\n",
+    "With an HRIR selected, we can plot it in the time and frequency domain."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ax = pf.plot.time_freq(hrirs[index], label=[\"Left ear\", \"Right ear\"])\n",
+    "ax[0].legend();\n",
+    "ax[1].legend();"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating a binaural synthesis\n",
+    "\n",
+    "The binaural synthesis is done by convolving a mono, anechoic signal with an HRIR.\n",
+    "The following code shows how to do this with `pyfar`.\n",
+    "We will create a function for this to reuse it later.\n",
```

Comment:
What do you think of moving the interactive section to a static notebook? With this, you could simplify here by not needing to define a function. The interactive section notebook could then reference here and define as a function.
The code block would be shortenened to
```
    # apply the HRIRs to the audio signal
    output_audio = pf.dsp.convolve(audio, hrirs[index])
    output_audio = 10 ** (gain / 20) * pf.dsp.normalize(output_audio)
```

Reply:
Good point. Let's talk on Friday.

Reply:
I'd suggest moving the entire notebook to a static version. The Audio output does still work in that case, since the Audio data is stored within the widget when executing the notebook.
And for the "interactive" experience, readers do need to open an interactive binder session anyways.